native/SampleApps/RestAPIExplorer/RestAPIExplorer/Classes/AppDelegate.m

```diff
@@ -48,6 +48,20 @@ - (id)init
     self = [super init];
     if (self) {
         [SFUserAccountManager sharedInstance].advancedAuthConfiguration = SFOAuthAdvancedAuthConfigurationRequire;
+        /*
+         //scheme of idpAppp
+         [SFUserAccountManager sharedInstance].advancedAuthConfiguration = SFOAuthAdvancedAuthConfigurationNone;
+         [SalesforceSDKManager sharedManager].idpAppURIScheme = @"sampleidpapp";
+         //user friendly display name
+         [SalesforceSDKManager sharedManager].appDisplayName = @"SampleAppOne";
+         
+         //Use the following code block to replace the login flow selection dialog
+         [SalesforceSDKManager sharedManager].idpLoginFlowSelectionBlock = ^UIViewController<SFSDKLoginFlowSelectionView> * _Nonnull{
+         IDPLoginNavViewController *controller = [[IDPLoginNavViewController alloc] init];
+         return controller;
+         };
+         */
+        
```

Comment:
Don't you need the commented out code from below here as well?

`return [[SFUserAccountManager sharedInstance] handleAdvancedAuthenticationResponse:url options:options];`

Reply:
@bhariharan This code is good. The RestApi Explorer has been set by default to handle advanced Auth. For the IDP case the call is exactly the same.

Reply:
Gotcha.